state-network.md

```diff
@@ -106,7 +106,7 @@ Merkle Patricia Trie (MPT) proofs consist of a list of witness nodes that corres
 
 ```
 WitnessNode            := ByteList(1024)
-MPTWitness             := List(witness: WitnessNode, max_length=32)
+MPTWitness             := List(witness: WitnessNode, max_length=1024)
```

Comment:
1024 seems like maybe overkill here for proof sizing but it might be nice to anchor this number to a real world analysis.